services/encrypt.js

```diff
@@ -21,5 +21,16 @@ function isValidPassword(user, password) {
   return bcrypt.compareSync(password, user.password);
 }
 
+/**
+ * Verifies whether the encrypted apikey is valid or not
+ * @param {Object} user
+ * @param {string} apikey
+ * @returns {boolean}
+ */
+function isValidAPIKey(user, apikey) {
+  return bcrypt.compareSync(apikey, user.apikey);
```

Comment:
We might want to go with `bcrypt.compare()` to keep from blocking other requests. I'm not sure why were doing `hashSync()`.

Reply:
Yeah good call, I'll dig into using the async methods instead.